src/components/VgtHeaderRow.vue

```diff
@@ -4,25 +4,51 @@
     v-if="headerRow.mode === 'span'"
     class="vgt-left-align vgt-row-header"
     :colspan="fullColspan"
-    @click="collapsable ? $emit('vgtExpand', !headerRow.vgtIsExpanded) : () => {}">
-    <span v-if="collapsable" class="triangle" :class="{ 'expand': headerRow.vgtIsExpanded }"></span>
-    <slot
-      :row="headerRow"
-      name="table-header-row">
-      <span v-if="headerRow.html" v-html="headerRow.label">
-      </span>
-      <span v-else>
-        {{ headerRow.label }}
-      </span>
-    </slot>
+    >
+    <template v-if="selectAllByGroup">
+      <slot name="table-header-group-select"
+        :columns="columns"
+        :row="headerRow"
+      >
+        <input
+          type="checkbox"
+          :checked="allSelected"
+          @change="toggleSelectGroup($event)" />
+      </slot>
+    </template>
+    <span @click="collapsable ? $emit('vgtExpand', !headerRow.vgtIsExpanded) : () => {}">
```

Comment:
I moved the @click to a span, that includes the triangle and the row label.
If we leave the @click on the <tr> then it would also expand when clicking the checkbox. Imho this was the best solution.